doc/user/content/transform-data/_index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Overview"
+description: "Learn to transform data using Materialize SQL."
+disable_list: true
+menu:
+  main:
+    parent: transform-data
+    weight: 5
+    identifier: transform-overview
+---
+
+With Materialize, organizations can use SQL to transform their data. Materialize
+follows the SQL-92 implementation, and in general, the implementation is modeled
+after PostgreSQL.
+
+### SELECT statement
+
+In Materialize, you can [`SELECT`](/sql/select/) from
+[sources](/concepts/sources/), tables, [views](/concepts/views/#views), and
+[materialized view](/concepts/views/#materialized-views).
+
+```mzsql
+SELECT [ ALL | DISTINCT [ ON ( col_ref [, ...] ) ] ]
+    [ { * | projection_expr [ [ AS ] output_name ] } [, ...] ]
+    [ FROM table_expr [ join_expr | , ] ... ]
+    [ WHERE condition_expr ]
+    [ GROUP BY grouping_expr [, ...] ]
+    [ OPTIONS ( option = val[, ...] ) ]
+    [ HAVING having_expr ]
+    [ ORDER BY projection_expr [ ASC | DESC ] [ NULLS FIRST | NULLS LAST ] [, ...] ]
+    [ LIMIT { integer  } [ OFFSET { integer } ] ]
+    [ { UNION | INTERSECT | EXCEPT } [ ALL | DISTINCT ] { SELECT ...} ]
+```
+
+In Materialize, the [`SELECT`](/sql/select/) statement supports (among others):
+
+- [JOINS (inner, left outer, right outer, full outer,
+  cross)](/sql/select/join/) and [lateral
+  subqueries](/sql/select/join/#lateral-subqueries)
+
+- [Common Table Expressions (CTEs)](/sql/select/#common-table-expressions-ctes)
+  and [Recursive CTEs](/sql/select/recursive-ctes/)
+
+- [Query hints (`AGGREGATE INPUT GROUP SIZE`, `DISTINCT ON INPUT GROUP SIZE`,
+  `LIMIT INPUT GROUP SIZE`)](/sql/select/#query-hints)
+
+- [SQL functions](/sql/functions/) and [operators](/sql/functions/#operators)
+
+For more information, see:
+
+- [`SELECT` reference page](/sql/select/)
+
+- [Query optimization](/transform-data/optimization/)
+
+### Views and materialized views
```

Comment:
Yup. Placeholder since we moved out from the menu the SQL reference pages - so, didn't want to strand people.

For best practices, it's either here or refashioned Patterns landing page.